java/ql/test/library-tests/dataflow/fluent-methods/flow.ql

```diff
@@ -0,0 +1,30 @@
+import java
+import semmle.code.java.dataflow.TaintTracking
+import TestUtilities.InlineExpectationsTest
+
+class Conf extends TaintTracking::Configuration {
```

Comment:
This entire test case (and the things it needs to test) are actually all data flow without any taint bits, so this should be a `DataFlow::Configuration` instead and the various references to "taint" should mostly just be "flow".